angularSTEP/src/app/recipe-card/recipe-card.component.html

```diff
@@ -1,12 +1,11 @@
-<mat-card class="example-card">
+<mat-card *ngIf='recipe' class="recipe-card">
+  <img mat-card-image *ngIf='recipe.images && recipe.images.length != 0' [src]='recipe.images[0]' alt="Photo of {{ recipe.recipeName }}" class="card-img">
+  <img mat-card-image *ngIf='recipe.images == null || recipe.images.length == 0' src="https://upload.wikimedia.org/wikipedia/commons/6/69/Banana.png" alt="Photo of stock food" class="card-img">
+  <mat-divider></mat-divider>
   <mat-card-header>
-    <mat-card-title>{{recipeName}}</mat-card-title>
-    <mat-card-subtitle>{{difficulty}}</mat-card-subtitle>
+      <mat-card-title>{{ recipeName }}</mat-card-title>
+      <mat-card-subtitle><span>{{recipe.difficulty}}</span>
+        <span class='rating'>{{ averageRating }}/5</span>
+      </mat-card-subtitle>
   </mat-card-header>
-  <img mat-card-image src="https://material.angular.io/assets/img/examples/shiba2.jpg" alt="Photo of a Shiba Inu">
-  <mat-card-content>
-    <p>
-      {{description}}
```

Comment:
I know you said you changed the recipe-card to better match the profile-card layout, but I think it'd be a good idea to have the description included in the recipe-card since it's purpose is to more or less help entice someone to click on the recipe to try. That way a user won't have to click on a recipe they're unsure of to make a decision based only to see the (short) description and if they go back to view all recipes or top recipes or wherever they previously were, the user won't have to find their place again.

You cited (at least I think you did but until now I thought recipe widget was different from recipe card (widget being multiple/where the cards would be hosted)) making the card smaller, so if that's the worry, then we could always make the character limit for the short description less than 200 characters. When I originally put 200 I didn't have an idea of how it might look on a card component since I wasn't working on that component and 200 didn't seem like too many or too little characters, so I have no problem with changing it to a lower limit if that would help the sizing issue.

Reply:
I forgot to add that besides this LGTM

Reply:
Okay I changed the profile card widget. Now it has a drop down to display the info and tags if you wanted to look at it before clicking on the recipe.

![image](https://user-images.githubusercontent.com/42589078/89189811-899b5a00-d56e-11ea-9afc-5ba28da7eef5.png)

![image](https://user-images.githubusercontent.com/42589078/89189823-8ef8a480-d56e-11ea-8086-5418b1916bfd.png)